openldap/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+  openldap:
+    image: osixia/openldap
+    volumes:
+      - ${PWD}/env:/container/environment/01-custom.yaml
```

Comment:
Test and see if `sample` is converted automatically to env file. If not, this needs to be addressed.